src/dev-starter.ts

```diff
@@ -66,14 +71,15 @@ export class DevStarter {
       console.error(chalk.red(data.toString()));
     });
 
-    return new Promise((resolve) => {
+    return new Promise<{ launchUrl: string; localhostUrl: string }>((resolve) => {
```

Comment:
Would `DevServerStartResult` be the correct type here?

```suggestion
    return new Promise<DevServerStartResult>((resolve) => {
```